math/OpenBLAS/Portfile

```diff
@@ -19,6 +20,9 @@ compiler.cxx_standard 2011
 compiler.c_standard   2011
 compiler.thread_local_storage yes
 
+# error: invalid instruction mnemonic 'cvtsi2ssl'
+compiler.blacklist-append {macports-clang-15}
```

Comment:
I'm surprised that this would be the fix for this issue. Can you add a URL with a bug report or more information confirming that this error is a bug in clang 15? The Trac ticket only talks about the `mktemp` problem.

Curly braces are not required here.

Reply:
@ryandesign I haven't find any bug report. I simple tried another clang (14) and it works :)

Yes, I've reproduced it on my machine before fixing it.